<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSettingsBase</class>
 <widget class="QWidget" name="ImageSettingsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KLineEdit" name="pathEdit"/>
     </item>
     <item>
      <widget class="KPushButton" name="openDialogButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Display Size</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="displayWidthInput"/>
      </item>
      <item row="0" column="2">
       <widget class="KComboBox" name="displayWidthCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="displayHeightInput"/>
      </item>
      <item row="1" column="2">
       <widget class="KComboBox" name="displayHeightCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="KImageFilePreview" name="imagePreview"/>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Print Size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="useDisplaySize">
        <property name="text">
         <string>Use Display Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="printWidthInput"/>
      </item>
      <item row="1" column="2">
       <widget class="KComboBox" name="printWidthCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="printHeightInput"/>
      </item>
      <item row="2" column="2">
       <widget class="KComboBox" name="printHeightCombo"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KImageFilePreview</class>
   <extends>QWidget</extends>
   <header>kimagefilepreview.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>openDialogButton</tabstop>
  <tabstop>displayWidthInput</tabstop>
  <tabstop>displayWidthCombo</tabstop>
  <tabstop>displayHeightInput</tabstop>
  <tabstop>displayHeightCombo</tabstop>
  <tabstop>printWidthInput</tabstop>
  <tabstop>printWidthCombo</tabstop>
  <tabstop>printHeightInput</tabstop>
  <tabstop>printHeightCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
